Tests/TracingInstrumentationTests/SpanTests.swift

```diff
@@ -183,6 +183,36 @@ final class SpanTests: XCTestCase {
         XCTAssertEqual(child.links[0].context[TestBaggageContextKey.self], "test")
         XCTAssertEqual(child.links[0].attributes.sampleHttp.statusCode, 418)
     }
+
+    func testRecordErrorDefaultsToCreatingSpanEvent() {
+        var span = OTelSpan(
```

Comment:
the rename?

Reply:
Rebased and force-pushed. I guess you were looking at an outdated version.